buildtest/cli/report.py

```diff
@@ -7,12 +7,29 @@
 from buildtest.defaults import BUILD_REPORT, BUILDTEST_REPORTS, console
 from buildtest.exceptions import BuildTestError
 from buildtest.utils.file import is_file, load_json, resolve_path
-from rich.color import Color, ColorParseError
+from rich.color import ANSI_COLOR_NAMES, Color, ColorParseError
 from rich.table import Table
 
 logger = logging.getLogger(__name__)
 
 
+def checkColor(colorArg):
+    """Checks the provided colorArg against the compatible colors from Rich.Color"""
+    if type(colorArg) is Color:
+        return colorArg.name
+
+    if colorArg and type(colorArg) is list:
```

Comment:
you can change this to `if isinstance(colorArg, list)`

Reply:
simplified